### PR TITLE
*: update docs about reset task and relay log purge

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- update the FAQ about reset task
- clarify the `purge-expires` config item

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### CODE REVIEW

File: zh/faq.md [hunk 1]
```diff
@@ -27,28 +27,16 @@ DM 会尝试将包含多个 DDL 变更操作的单条语句拆分成只包含一
...
-
-    - 如果是使用 DM-Ansible 部署，relay log 目录即 `<deploy_dir>/relay_log` 目录。
-    - 如果是使用二进制文件手动部署，relay log 目录即 relay-dir 参数设置的目录。
-
-4. 清理掉下游已同步的数据。
-
-5. 使用 DM-Ansible [启动整个 DM 集群](deploy-a-dm-cluster-using-ansible.md#第-9-步部署-dm-集群)。
-
-6. 以新的任务名重启数据同步任务，或设置 `remove-meta` 为 `true` 且 `task-mode` 为 `all`。
+    - 修改任务配置文件以指定新的任务名，然后使用 `start-task {task-config-file}` 重启同步任务。
```
[the remark this answers is not in the document]
Comment (csuzhangxc) [reply zh/faq.md:38]: 这里想表达的，其实  通过修改配置文件的方式  来  指定一个新的任务名。

File: zh/relay-log.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ DM-worker 每次启动时（或在 DM-worker 暂停后 relay log 恢复同步）
 
 ## 数据清理
 
-因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前任务稍后会使用到的 relay log。
+因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前已有数据迁移任务稍后会使用到的 relay log。
```
[the remark this answers is not in the document]
Comment (csuzhangxc) [reply zh/relay-log.md:90]: 直接用 `之后`/`以后`？

File: zh/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (toutdesuite) [reply zh/faq.md:38]: ok

File: zh/relay-log.md (hunk 2 above)
Comment (toutdesuite) [inline zh/relay-log.md:90]:
```suggestion
因为存在文件读写的检测机制，所以 DM-worker 不会清理正在使用的 relay log，也不会清理当前已有数据迁移任务之后会使用到的 relay log。
```

**Pull request merged**